browser/main/NoteList/index.js

```diff
@@ -163,6 +166,32 @@ class NoteList extends React.Component {
     ee.emit('list:moved')
   }
 
+  jumpNoteByHashHandler (event, noteHash) {
```

Comment:
You can do it by cut out this logic to the module.
Like this.

```javascript
export function jumpToNote(event, noteHash, router, location) {
  // brabra
}
```

You can use it on any other components.